data/frameSimulator/src/checkframe.py

```diff
@@ -0,0 +1,68 @@
+#!/dls_sw/apps/python/anaconda/1.7.0/64/bin/python
+
+import time
+import os
+import re
+import struct
+import re;
+import h5py;
+import numpy as np;
+
+
+
+f = h5py.File('/tmp/haha_000001.h5','r')
```

Comment:
What happens when this file does not exist?  Should this be a command line option?